includes/polling/cisco-nac.inc.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+use LibreNMS\Util\IP;
+
+if (!get_dev_attrib($device, 'poll_cisco-nac', 'true')) {
+    echo "\nCisco-NAC\n";
+    $ports_mapped = get_ports_mapped($device['device_id'], true);
+    $ports = $ports_mapped['ports'];
+    $PortAuthSessionEntry = snmpwalk_cache_oid($device, 'cafSessionEntry', array(), 'CISCO-AUTH-FRAMEWORK-MIB');
+    $cafSessionMethodsInfoEntry = snmpwalk_cache_oid($device, 'cafSessionMethodsInfoEntry', array(), 'CISCO-AUTH-FRAMEWORK-MIB');
+    foreach ($PortAuthSessionEntry as $PortAuthSessionEntryNAC => $PortAuthSessionEntryParameters) {
+        $port_index_nac = substr($PortAuthSessionEntryNAC, 0, strpos($PortAuthSessionEntryNAC, "."));
+        $port_auth_id_nac = strstr($PortAuthSessionEntryNAC, "'");
+        $port_auth_id_nac = substr($port_auth_id_nac, 0, -1);
+        $port_auth_id_nac = substr($port_auth_id_nac, 1);
+        $CheckExistIndex = dbFetchRow('SELECT * FROM `ports_nac` WHERE `port_index` = ?', $port_index_nac);
```

Comment:
Still missing schema file to define ports_nac table.

Reply:
Done